src/Scripting/CSharpTest.Desktop/CsiTests.cs

```diff
@@ -47,7 +48,17 @@ public void CurrentWorkingDirectory_Change()
 new C()
 Environment.Exit(0)
 ";
-            var result = ProcessUtilities.Run(CsiPath, "", stdInput: stdInput);
+            ProcessResult result = null;
```

Comment:
I would set the encoding to UTF8 without BOM, so that we can test non-ascii input. ```new UTF8Encoding(encoderShouldEmitUTF8Identifier: false)```.